src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/UpdateApprenticeshipStatus/UpdateApprenticeshipStatusCommandHandler.cs

```diff
@@ -71,28 +71,19 @@ protected override async Task HandleCore(UpdateApprenticeshipStatusCommand comma
         {
             if (command.ChangeType == ChangeStatusType.Stop) // Only need to validate date for stop currently
             {
-                if (apprenticeship.IsWaitingToStart(_currentDateTime))
```

Comment:
Why have we removed this rule ?

This still seems relevant?

Reply:
we dont need this rule to validate for the status WaitingToStart. The start date would be always the future date and the date change would be the current date.